src/sdk/data/data.module.ts

```diff
@@ -249,4 +249,65 @@ export class DataModule {
       throw new Error(error.message || 'Failed to fetch exchange rates');
     }
   }
+
+  async getConnextSupportedAssets(chainId?: number): Promise<ConnextToken[]> {
```

Comment:
I would have a single endpoint for any underlying provider. call it `getSupportedAssets` passing in an optional Provider (with default value as Connext).

Reply:
I have made this change.